.github/workflows/fondue-pull-requests.yaml

```diff
@@ -112,11 +112,18 @@ jobs:
             - name: Setup pnpm cache
               uses: actions/cache@v3
               with:
-                  path: |
-                      ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
-                      ~/.cache/Cypress
-                  key: pnpm-with-cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
-                  restore-keys: pnpm-with-cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
```

Comment:
splitting the 2 cache operations separately to be more explicit and matching the new component workflow, where we are not caching the entire node_modules directory.